### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by SeanHH86**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="left">
-    <a href="README_EN.md">English</a> ｜ 中文
+    <a href="https://github.com/OpenCSGs/csghub-sdk/blob/main/README_EN.md">English</a> ｜ 中文
```
Comment (Rader) [inline README.md:2]: why use absolute path?

File: README_EN.md [hunk 2]
```diff
@@ -149,6 +149,19 @@ cache_dir = '/home/test/'
...
+
+```python
+from pycsghub.file_upload import http_upload_file
+
+token = "3b77c98077b415ca381ded189b86d5df226e3776"
+
+endpoint = "https://hub.opencsg.com"
+repo_type = "model"
+repo_id = 'wanghh2000/myprivate1'
+result = http_upload_file(repo_id, endpoint=endpoint, token=token, repo_type='model', file_path='test1.txt')
```
Comment (Rader) [inline README_EN.md:162]: does sdk support uploading multiple files once? any example code?

File: README.md (hunk 1 above)
Comment (SeanHH86) [reply README.md:2]: pypi cannot switch readme, so link english version to github.

File: README_EN.md (hunk 2 above)
Comment (SeanHH86) [reply README_EN.md:162]: It is easy upload files in loop, append use case code

**Review by Rader — APPROVED** [review]

**Comment by SeanHH86** [comment]
#27

**Pull request merged**